qc_grader/challenges/quantum_explorers23/__init__.py

```diff
@@ -25,3 +25,24 @@
 from .badge_3 import (
     grade_badge3_ex1
 )
+
+
+from .badge_5 import (
+    grade_badge5_ex1,
+    grade_badge5_ex2,
+    grade_badge5_ex3,
+    grade_badge5_ex4,
+    grade_badge5_ex5,
+    grade_badge5_ex6,
+    grade_badge5_ex7,
+    grade_badge5_ex8,
+    grade_badge5_ex9,
+    grade_badge5_ex10,
+    grade_badge5_ex11,
+    grade_badge5_ex12,
+    grade_badge5_ex13,
+    grade_badge5_ex14,
+    grade_badge5_ex15,
```

Comment:
these two (`grade_badge5_ex14` & `grade_badge5_ex15`) do not exist in the `badge_5.py` file

Reply:
@vabarbosa yes, thank you. added the missing graders to badge5.py